metadata/kmwversions.json

```diff
@@ -0,0 +1,2089 @@
+{
+    "versions": [
+        "17.0.132",
```

Comment:
Seems this list contains all version numbers twice...

Reply:
good catch! I've tweaked to hopefully fix that